### ISSUE TITLE
validation accuracy

### ISSUE THREAD

**Issue by KentaItakura** [matlab-deep-learning/pretrained-efficientdet-d0#2]
Hi, thank you so much for your hard work for efficientDet. 

Is it possible for you to update code for computing validation loss like YOLOv2 example in matlab documentation? The current version splits the vehicle data into training and test dataset, but I want you to split into train, valid and test.
